redaxo/src/addons/install/lib/command/download.php

```diff
@@ -37,7 +37,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if (null === $version) {
             $versions = [];
             foreach ($files as $fId => $fileMeta) {
-                $versions[$fId] = $fileMeta['version'];
+                $versions[] = $fileMeta['version'];
             }
 
             $version = $io->choice('Please choose a version', $versions);
```

Comment:
Musst du hier jetzt rückwärts mappen auf die interne id?

Reply:
brauch ich nicht, weil in `$version = $io->choice()` die Versionsnummer (z.B. 3.8.0) landet. Im Anschluss greift die selbe Logik wie für das Console Argument